pkgs/games/r2mod_cli/default.nix

```diff
@@ -0,0 +1,45 @@
+{ fetchFromGitHub
+, jq
+, makeWrapper
+, p7zip
+, stdenv
+}:
+
+stdenv.mkDerivation rec {
+  pname = "r2mod_cli";
+  version = "1.0.5";
+
+  src = fetchFromGitHub {
+    owner = "Foldex";
+    repo = "r2mod_cli";
+    rev = "v${version}";
+    sha256 = "1g64f8ms7yz4rzm6xb93agc08kh9sbwkhvq35dpfhvi6v59j3n5m";
+
+  };
+
+  buildInputs = [
+    jq
+    makeWrapper
+    p7zip
```

Comment:
You shouldn't need `p7zip` neither `jq` here.

Reply:
Sure enough.